client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -78,35 +92,41 @@ export const PatternsCategoryPage = ( {
 		}
 	}, [ category ] );
 
+	const isHomePage = ! category && ! searchTerm;
+
 	const categoryObject = categories?.find( ( { name } ) => name === category );
 
 	const categoryNavList = categories?.map( ( category ) => {
 		const patternTypeFilterFallback =
 			category.pagePatternCount === 0 ? PatternTypeFilter.REGULAR : patternTypeFilter;
 
 		return {
-			name: category.name || '',
+			name: category.name,
 			label: category.label,
 			link: getCategoryUrlPath( category.name, patternTypeFilterFallback, false ),
 		};
 	} );
 
 	return (
 		<>
-			<DocumentHead title="WordPress Patterns- Category" />
+			<DocumentHead title="WordPress Patterns - Category" />
 
 			<PatternsHeader
-				description="Introduce yourself or your brand to visitors."
+				description={
+					category
+						? 'Introduce yourself or your brand to visitors.'
+						: 'Hundreds of expertly designed, fully responsive patterns allow you to craft a beautiful site in minutes.'
+				}
```

Comment:
This is slightly messy at the moment, but this logic is all due to change when we address https://github.com/Automattic/dotcom-forge/issues/5701

Reply:
It's also tiny, but one more point to work with this refactoring https://github.com/Automattic/dotcom-forge/issues/6110
Since theoretically we can have some cases with ternary operator for search / category / home